actions.js

```diff
@@ -10,7 +10,8 @@ var twitterAPI = require('node-twitter-api'),
     Q = require('q'),
     _ = require('sequelize').Utils._,
     util = require('./util'),
-    setup = require('./setup');
+    setup = require('./setup'),
+    verifyCredentials = require('./verify-credentials');
```

Comment:
In retrospect, it might be good to do a project wide search for `.verifyCredentials` being used on a btuser object.